scippy/src/scippy/plot.py

```diff
@@ -347,37 +382,124 @@ def plot_image(input_data, name=None, axes=None, contours=False, cb=None,
 
 class ImageViewer:
 
-    def __init__(self, xe, xc, ye, yc, z, resolution, cb, plot_type, title,
-                 contours):
+    def __init__(self, xe, xc, ye, yc, values, variances, resolution, cb,
+                 plot_type, title, contours):
 
         self.xe = xe
         self.xc = xc
         self.ye = ye
         self.yc = yc
-        self.z = z
+        self.values = values
+        self.variances = variances
         self.resolution = resolution
         self.cb = cb
         self.plot_type = plot_type
         self.title = title
         self.contours = contours
         self.nx = len(self.xe)
         self.ny = len(self.ye)
+        self.val_range = np.zeros([4], dtype=np.float64)
+        self.var_range = np.zeros([4], dtype=np.float64)
 
-        self.fig = FigureWidget()
+        self.fig_val = FigureWidget()
+        if self.variances is not None:
+            self.fig_var = FigureWidget()
+            self.hbox = HBox([self.fig_val, self.fig_var])
+        else:
+            self.fig_var = None
+            self.hbox = None
 
         # Make an initial low-resolution sampling of the image for plotting
-        self.resample_image(layout=None, x_range=[self.xe[0], self.xe[-1]],
-                            y_range=[self.ye[0], self.ye[-1]])
+        self.update_values(layout=None, x_range=[self.xe[0], self.xe[-1]],
+                           y_range=[self.ye[0], self.ye[-1]])
 
         # Add a callback to update the view area
-        self.fig.layout.on_change(
-            self.resample_image,
+        self.fig_val.layout.on_change(
+            self.update_values,
             'xaxis.range',
             'yaxis.range')
+        if self.variances is not None:
```

Comment:
I think in the majority of cases you would not want to plot variances, my suggestion was that it would be nice to have the option if one wished. I suggest that the default is not to plot the variance side-by-side but make it something that can be enabled.